code/modules/clothing/clothing.dm

```diff
@@ -31,6 +31,10 @@
 
 	var/move_trail = /obj/effect/decal/cleanable/blood/tracks/footprints
 
+	var/obj/item/clothing/head/winterhood/hood
+	var/hoodtype = null
+	var/clothing_type // used for holsters & hoods. Do not set it outside init.
+	var/hooded = FALSE
```

Comment:
This is too hardcoded, a better way of handling this would be to make a hood that registers signals on its parent, listening to the procs assigned below, and reacting accordingly.